gtfs_realtime_translators/translators/wcdot_bus.py

```diff
@@ -0,0 +1,60 @@
+import json
+from gtfs_realtime_translators.factories import TripUpdate, FeedMessage
+
+
+class WcdotGtfsRealTimeTranslator:
+    def __init__(self, stop_id=None):
+        if stop_id is None:
+            raise ValueError('stop_id is required.')
+        self.stop_id = stop_id
+        self.filtered_stops = None
+
+    def __call__(self,data):
+        json_data = json.loads(data)
+        entities = json_data["entity"]
+        trip_updates = generate_trip_updates(entities)
+        return FeedMessage.create(entities=trip_updates)
+        
+    def generate_trip_updates(entities):
+        trip_updates = []
+        for entity in entities:
+            current_trip_update = {}
+            current_trip_update['entity_id'] = entity['id']
+            process_trip_update(entity['trip_update'],current_trip_update)
```

Comment:
I am also not sure you can modify the passed object in place and would not recommend it. Best to return a new object